public/css/user.css

```diff
@@ -79,4 +79,71 @@
         font-family: 'Jomhuria', cursive;
     }
     
-    
\ No newline at end of file
+    .userq{
+        background-image: url('../img/background.png');
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover;
+        height: 549px;
+        overflow: auto;
+        position: relative;
+    }
+    .userq-contanier{
+        padding: 10px;
+        float: left;
+        margin-top: 40px;
+        margin-left: 50px;
+        margin-bottom: 40px;
+        height: 430px;
+        width: 54%;
+        min-width: 300px;;
+        background-color: #00000060;
+        border-radius: 22px;
+        box-shadow: 3px 4px 0px #00000025, 2px 3px 3px #737373;
+        display: flex;
+        flex-direction: column;
+    }
+    .userq-title{
+        font-size: 40px;
+        color: #333333;
+        margin: 20px 50px 20px 50px;
+        font-family: 'Jomhuria', cursive;
+    }
+    .userq-form{
+        margin: 20px 50px 20px 50px;
```

Comment:
@naremanhilles You could shorten this to `margin: 20px 50px;` if you want to save a few characters 😄

Reply:
thank you tom,i edit it :+1: